### PR TITLE
[REG-1213] Fix docs from play test

### PR DESCRIPTION
**Pull request by vontell**
Adds fixes for the following doc issues discovered last week:

* Fixes a few types
* Adds a new RGFindUtils page
* Updates language in a few places to make the tutorial more clear for more use cases
* Fixes Git link to be usable in Unity
* Mentions defaults more clearly

### CODE REVIEW

File: docs/studios/unity/unity-sdk/RGFindUtils.mdx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+# RGFindUtils
+
+`RGFindUtils` is a singleton providing a set of utility functions for finding GameObjects. This
+is extremely useful when implementing [`RGAction`](./RGAction)s that need to find other GameObjects in the scene.
+
+The instance of `RGFindUtils` is automatically created when the scene is loaded, and can be accessed
+with the following function:
+
+```cs
+RGFindUtils.Instance
```
Comment (abeizer) [inline docs/studios/unity/unity-sdk/RGFindUtils.mdx:14]: Not really a not for these docs, but in general - some of our singletons follow this pattern while others are accessed using a `GetInstance()` method. We should consider making these consistent

File: docs/studios/unity/unity-sdk/RGAction.mdx [hunk 2]
```diff
@@ -8,6 +8,9 @@ Represents an action that a bot can instruct the game object to take.
 This `RGAction` class should be inherited in a sub class, and developers
 will need to implement the `StartAction` and `GetActionName` function.
 
+Also see [`RGFindUtils`](./RGFindUtils) and [`RGState`](./RGState) for concepts that come up when
```
Comment (abeizer) [inline docs/studios/unity/unity-sdk/RGAction.mdx:11]: If we're telling people to reference another page to "find out how to do x", then it'll need to be more specific. I really like the way you worded it in the first_tutorial.md 

> See the [`RGFindUtils`](../unity-sdk/RGFindUtils) class for helper functions which find GameObjects and components within your scene based on information coming in from an action.

**Review by abeizer — CHANGES_REQUESTED** [review]

File: docs/studios/unity/unity-sdk/RGFindUtils.mdx (hunk 1 above)
Comment (vontell) [reply docs/studios/unity/unity-sdk/RGFindUtils.mdx:14]: Yeah agreed... I'll make sure to write that down somewhere! @rcornwall-reg what do you find is most common? Or I think maybe you even mentioned not going with a singleton and going with a different approach?

File: docs/studios/unity/unity-sdk/RGAction.mdx (hunk 2 above)
Comment (vontell) [reply docs/studios/unity/unity-sdk/RGAction.mdx]: Good suggestion! Let me know what you think of my edit.

Comment (abeizer) [reply docs/studios/unity/unity-sdk/RGAction.mdx]: I'd remove the bit about "an easier way" because it's not really a separate method of defining actions, just helpers to do so. It'd also be weird if we were presenting them with "the hard way" first.

Suggest rewording to something like "In most RGAction implementations, it is necessary to find game objects within the scene based on the game state. See the RGState class for defining the game state and the RGFindUtils class for helper functions to locate GameObjects within your scene."

Comment (vontell) [reply docs/studios/unity/unity-sdk/RGAction.mdx]: Added!

**Review by abeizer — APPROVED** [review]

**Review by RG-nAmKcAz — APPROVED** [review]
